packages/x-date-pickers/src/DatePicker/DatePicker.tsx

```diff
@@ -186,6 +186,10 @@ DatePicker.propTypes = {
    * @deprecated Use the `localeText` prop of `LocalizationProvider` instead, see https://mui.com/x/react-date-pickers/localization
    */
   getViewSwitchingButtonText: PropTypes.func,
+  /**
```

Comment:
Thank you for the contribution.
Proptypes in our project are generated as are their comments.
Could you please update the comment on this corresponding field in `PureDateInput`, run `yarn proptypes` and then commit the changes?